examples/sri-1/components/workgroup/invitation/invitation.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { Agreement } from "../storage/agreement";
+import { BpiSubject } from "../identity/bpiSubject";
+export interface IInvitation {
+    id: string;
+    name: string;
+    sender: BpiSubject;
+    recipient: string;
+    workgroupId: string;
```

Comment:
same as above

Reply:
TODOs added from id to uuid change at #491